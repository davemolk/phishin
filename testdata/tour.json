{"success":true,"total_entries":1,"total_pages":1,"page":1,"data":{"id":3,"name":"1985 Tour","shows_count":6,"slug":"1985-tour","starts_on":"1985-03-04","ends_on":"1985-11-23","shows":[{"id":3,"date":"1985-03-04","duration":2414471,"incomplete":true,"sbd":true,"remastered":false,"tour_id":3,"venue_id":326,"likes_count":20,"taper_notes":"Phish\r\nMarch 4, 1985\r\nHunt's\r\nBurlington, VT\r\n\r\nSOURCE: SBD \u003e Cass0 \u003e DAT\r\nTRANSFER: Archive Python \u003e vDAT \u003e Wavelab \u003e FLAC16\r\nThanks to Ryan Mann for the source DAT!\r\nTransfer, mastering, and FLAC16 by: Marmar- imthemarmar@gmail.com\r\n\r\n01. Anarchy\r\n02. Camel Walk\r\n03. Fire Up the Ganja\r\n04. Skippy the Wondermouse\r\n05. In the Midnight Hour\r\n\r\nShow Notes: \r\n- Fire up the Ganja featured members of the band Lamb's Bread.","updated_at":"2018-12-21T08:10:12Z","venue_name":"Hunt's","location":"Burlington, VT"}]}}