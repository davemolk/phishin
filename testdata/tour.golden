Name:      Starts On:  Ends On:    Show Count:
1985 Tour  1985-03-04  1985-11-23  6

ID:  Date:       Venue:  Location:       Duration:  Soundboard:  Remastered:
3    1985-03-04  Hunt's  Burlington, VT  40m 14s    yes          no
